packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -195,6 +206,14 @@ const AppModals = observer(() => {
         ComponentToLoad = <UrlUnavailableModal />;
     }
 
+    if (has_wallet && should_show_wallets_upgrade_completed_modal) {
+        ComponentToLoad = <WalletsUpgradeCompletedModal />;
+    }
+
+    if (!has_wallet && is_migrated) {
+        ComponentToLoad = <WalletsUpgradeLogoutModal />;
+    }
```

Comment:
how it works together with other modals selection logic? whats the logic behind the order and priorities of selecting the modals to display?
especially, should it be lower on the list, given its immediate a result of user action (migration) and thus should have priority over pretty much anythign else?

Reply:
Yes, the priority of the core modals is not clear / explicit, so I had to follow my own judgement on where to place the if-clause

Reply:
There are actually 2 priority chains there in the file one after the other
- descending priority if-elseif chain
- ascending priority ifs sequence
This probably needs some standardization